### ISSUE TITLE
SetIcon for dialogs

### ISSUE THREAD

**Issue by vomikan** [moneymanagerex/moneymanagerex#5592]
Should we change the dialog icon to the corresponding icon?

After:
![image](https://user-images.githubusercontent.com/6836805/215322621-83c17d9e-b000-42a3-9dbe-e0f9187e1b85.png)

Before:
![image](https://user-images.githubusercontent.com/6836805/215322765-aae8bf63-da6b-4d7e-912b-2a92205f44bd.png)

**Comment by renato-mmex** [comment]
and where is the difference

**Comment by vomikan** [comment]
> and where is the difference

![image](https://user-images.githubusercontent.com/6836805/215327974-448d887d-edc9-46f9-89af-a94960cb4c2d.png)

**Comment by tactilis** [comment]
MMEX dialogs currently have two title bar icon styles.

1. The majority have the MMEX application icon in the title bar, for example:
 
![image](https://user-images.githubusercontent.com/4060092/215336061-e765684b-ae80-474d-ae3b-43f16033fc41.png)
 
2. A few have no icon in the title bar, for example:
 
![image](https://user-images.githubusercontent.com/4060092/215336092-f78c1928-636a-4144-96f7-09dff1a6f4ca.png)
 
As there are only a few dialogs that could be changed to have a task specific icon (like you are proposing for Transaction Filter), my view is that we should be consistent use the MMEX application icon everywhere.

**Comment by rmelillo76** [comment]
I like this idea, to me it helps give MMEX a more polished look and feel. I know earlier versions of MMEX ( < 1.4) had icons in the menus, I wonder if there is any desire to bring that back and incorporate it into this idea as well.

![image](https://user-images.githubusercontent.com/95182117/215337729-c0fcba87-f18a-4596-a689-a2d26137fb69.png)

**Comment by tactilis** [comment]
> to me it helps give MMEX a more polished look and feel.

To me it makes MMEX look cluttered and dated.

**Comment by rmelillo76** [comment]
> > to me it helps give MMEX a more polished look and feel.
> 
> To me it makes MMEX look cluttered and dated.

Fair enough, this is a personal preference, and my statement about a polished look and feel was in response to vomikan's dialog icon title suggestion which is obviously not cluttered and dated. It sounds like are you referring to the menu icons, which was a secondary point that I merely brought up for folks to consider, that's all.

**Comment by whalley** [comment]
> As there are only a few dialogs that could be changed to have a task specific icon (like you are proposing for Transaction Filter), my view is that we should be consistent and use just the MMEX application icon everywhere.

Agreed, let's go for a common feel acrosss all dialogs
